src/storage/statistics_cl.c

```diff
@@ -150,6 +152,11 @@ stats_client_unpack_statistics (char *buf_p)
       attr_stats_p->n_btstats = OR_GET_INT (buf_p);
       buf_p += OR_INT_SIZE;
 
+      /*
+         OR_GET_INT64 (buf_p, &attr_stats_p->ndv);
+         buf_p += OR_INT64_SIZE;
+       */
+
```

Comment:
If there is no reason to leave it as a comment, please clear it.

Reply:
it's related to next PR. I keep it for next PR.